tests/sentry/web/frontend/test_auth_login.py

```diff
@@ -222,15 +232,20 @@ def test_registration_single_org(self):
         user = User.objects.get(username="test-a-really-long-email-address@example.com")
 
         # User is part of the default org
-        default_org = Organization.get_default()
-        org_member = OrganizationMember.objects.get(organization_id=default_org.id, user_id=user.id)
+        with assume_test_silo_mode(SiloMode.REGION):
+            default_org = Organization.get_default()
+        with assume_test_silo_mode(SiloMode.REGION):
+            org_member = OrganizationMember.objects.get(
```

Comment:
```suggestion
        with assume_test_silo_mode(SiloMode.REGION):
            default_org = Organization.get_default()
            org_member = OrganizationMember.objects.get(
```

Should we combine these?